README.md

```diff
@@ -1,46 +1,27 @@
-# telliot-feed-examples
-
-## Setup
-
-1. navigate to your home directory:
-    ```
-    cd ~/
-    ```
-2. get the code:
-    ```
-    git clone https://github.com/tellor-io/telliot-feed-examples.git
-    ```
-3. install python 3.8
-    - if using ubuntu 18 or 20:
-    ```
-    sudo apt-get update
-    sudo apt-get install python3.8 python3-pip
-    ```
-4. create virtual environment:
-    ```
-    python3 -m venv env
-    source env/bin/activate
-    ```
-5. install dependencies:
-    ```
-    pip3 install -r requirements-dev.txt
-    ```
-6. generate default `telliot` configs:
-    ```
-    telliot config init
-    ```
-7. generate default AMPL configs:
-    ```
+# Telliot Feed Examples
+
+## Project documentation:
+
+- [Telliot Feed Examples](https://tellor-io.github.io/telliot-feed-examples/)
+
+## Installation and Configuration:
+
+- [Getting Started](https://tellor-io.github.io/telliot-feed-examples/getting-started/)
+
+## Report legacy id 41 on Rinkeby
+
+To report the legacy request id 41 price on Rinkeby:
+
+1. Make sure that Telliot is configured for Rinkeby (`chain_id=4`)
+
+2. Enter the command line:
+
+   telliot-examples report legacyid 41
```

Comment:
`telliot-examples report uspce`

Reply:
Not supported by the `report legacyid` subcommand because it requires manual input, but I can change that.

Reply:
Yea, I just took a guess here.  Go ahead and update to whatever the syntax is.